requirements.txt

```
@@ -84,3 +84,6 @@ chardet==3.0.4
 django-admin-json-editor==0.2.3
 django-tinymce==3.4.0
 django-simple-history==3.0.0
+
+# Shapefile manipulation
+pyshp==2.3.1
```

```suggestion
pyshp==2.3.1

```
Needs a new line at the end of file